### PR TITLE
Map Popup show company name in all map popup titles

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
It was a little confusing that there was no title in the station names. Especially when feeds had terrible station names. We resolve this. As a result we remove company name from the other strings. However, the car isn't localized, so it's kept for now. In the future we might want to get rid of that

### CODE REVIEW

**Comment by miles-grant-ibigroup** [comment]
@daniel-heppner-ibigroup I need your help updating the snapshtos

**Review by daniel-heppner-ibigroup — APPROVED** [review]
Good change!

File: packages/map-popup/i18n/en-US.yml [hunk 1]
```diff
@@ -3,7 +3,8 @@ otpUi:
     availableBikes: "Available bikes: {value}"
     availableDocks: "Available docks: {value}"
     floatingBike: "Free-floating bike: {name}"
-    floatingCar: "{company} {name}"
+    floatingCar: "Car: {name}"
```
Comment (binh-dam-ibigroup) [inline packages/map-popup/i18n/en-US.yml:6]: Should this change apply to other languages too?

File: packages/map-popup/i18n/fr.yml [hunk 2]
```diff
@@ -5,5 +5,6 @@ otpUi:
     floatingBike: "Vélo flottant : {name}"
     floatingCar: "{company} {name}"
     floatingEScooter: Trottinette {name}
+    popupTitle: "{stationNetwork, select, false {{name}} other {{stationNetwork} {name}}}"
```
Comment (binh-dam-ibigroup) [inline packages/map-popup/i18n/fr.yml:8]: If `popupTitle` is going to be telegraphic style at first, does it need to be in the language files?

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
I think you need to apply the "Car" labeling to the other languages.

File: packages/map-popup/i18n/en-US.yml (hunk 1 above)
Comment (miles-grant-ibigroup) [reply packages/map-popup/i18n/en-US.yml:6]: I made the change backwards compatible so I don't have to make all the translations. I don't feel confident translating for all those languages

File: packages/map-popup/i18n/fr.yml (hunk 2 above)
Comment (miles-grant-ibigroup) [reply packages/map-popup/i18n/fr.yml:8]: Same as above. I've made it backwards compatible so languages can adjust when they're ready. Should we consider removing the strings from other languages?

File: packages/map-popup/i18n/en-US.yml (hunk 1 above)
Comment (binh-dam-ibigroup) [inline packages/map-popup/i18n/en-US.yml:6]: Is the colon needed?

Comment (miles-grant-ibigroup) [reply packages/map-popup/i18n/en-US.yml:6]: Without the colon things can get messy especially if the company name is silly

File: packages/map-popup/src/MapPopup.story.tsx
Comment (binh-dam-ibigroup) [inline packages/map-popup/src/MapPopup.story.tsx:1]: Add a story for the car case, for instance:
```js
export const FloatingCarEntity = (): JSX.Element => (
  <MapPopupContents entity={FLOATING_CAR} setLocation={action("setLocation")} />
);
```
where
```js
const FLOATING_CAR = {
  "stroke-width": 1,
  allowDropoff: false,
  allowPickup: true,
  color: "#f00",
  id: "car_6861",
  isCarStation: false,
  isFloatingCar: true,
  name: "0541",
  networks: ["CARONDEMAND"],
  realTimeData: true,
  spacesAvailable: 0,
  x: -122.70486,
  y: 45.525486666666666
};
```

**Review by binh-dam-ibigroup — APPROVED** [review]
See if you can add the additional story to illustrate the updated floating car case. Otherwise I am good with the changes.

File: packages/map-popup/src/MapPopup.story.tsx
Comment (miles-grant-ibigroup) [reply packages/map-popup/src/MapPopup.story.tsx:1]: Good call! Added

**Comment by miles-grant-ibigroup** [comment]
@daniel-heppner-ibigroup I'm going to need you to please update the snapshots again

**Pull request merged**